src/test/csi-neonsan/templates/node-ds.yaml

```diff
@@ -0,0 +1,69 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-neonsan-node
+  namespace: {{ .Release.Namespace }}
+  labels:
+    role: "node"
+spec:
+  selector:
+    matchLabels:
+      role: "node"
+  template:
+    metadata:
+      labels:
+        role: "node"
+    spec:
+      serviceAccount: csi-neonsan-node
+      tolerations:
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - key: dedicated
+        operator: Exists
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+      - effect: NoSchedule
+        key: node.cloudprovider.kubernetes.io/uninitialized
+        value: "true"
+      hostNetwork: true
+      containers:
+        - name: node-registrar
```

Comment:
Shall we need csi-neonsan container in this daemonset?

Reply:
neonsan-plugin run as systemd service, so csi-neonsan not need in the container

Reply:
OK